tests/integration/targets/aci_endpoint_ip_tag/tasks/main.yml

```diff
@@ -0,0 +1,418 @@
+# Test code for the ACI modules
+# Copyright: (c) 2025, Sabari Jaganathan (@sajagana)
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an ACI APIC host, ACI username and ACI password
+  ansible.builtin.fail:
+    msg: "Please define the following variables: aci_hostname, aci_username and aci_password."
+  when: aci_hostname is not defined or aci_username is not defined or aci_password is not defined
+
+# SET VARS
+- name: Set vars
+  ansible.builtin.set_fact:
+    aci_info: &aci_info
+      host: "{{ aci_hostname }}"
+      username: "{{ aci_username }}"
+      password: "{{ aci_password }}"
+      validate_certs: "{{ aci_validate_certs | default(false) }}"
+      use_ssl: "{{ aci_use_ssl | default(true) }}"
+      use_proxy: "{{ aci_use_proxy | default(true) }}"
+      output_level: '{{ aci_output_level | default("info") }}'
+
+- name: Query system information
+  cisco.aci.aci_system:
+    <<: *aci_info
+    id: 1
+    state: query
+  register: version
+
+- name: Ensure ansible_test tenant exists
+  cisco.aci.aci_tenant:
+    <<: *aci_info
+    tenant: ansible_test
+    state: present
+
+- name: Ensure ansible_test_vrf VRF exists
+  cisco.aci.aci_vrf:
+    <<: *aci_info
+    tenant: ansible_test
+    vrf: ansible_test_vrf
+    state: present
+
+- name: Query all existing Endpoint IP Tags
+  cisco.aci.aci_endpoint_ip_tag:
+    <<: *aci_info
+    state: query
+  register: query_all_ip_tags
+
+- name: Assertion check for query all existing Endpoint IP Tags
+  ansible.builtin.assert:
+    that:
+      - query_all_ip_tags is not changed
+
+- name: Clear existing Endpoint IP Tags from ansible_test tenant
+  cisco.aci.aci_endpoint_ip_tag:
+    <<: *aci_info
+    tenant: ansible_test
+    endpoint_ip_address: "{{ item.fvEpIpTag.attributes.ip }}"
+    vrf: "{{ item.fvEpIpTag.attributes.ctxName }}"
+    state: absent
+  when: "{{ item.fvEpIpTag.attributes.dn is match('uni/tn-ansible_test/.+?')  }}"
+  loop: "{{ query_all_ip_tags.current | list }}"
```

Comment:
do we need this or can we delete tenant before creation in test init? since we check query_all_ip_tags_1.current | length >= 2 later on so it does not matter if there are dangling in other tenants